src/eda_server/project.py

```diff
@@ -375,3 +316,238 @@ async def tar_project(db: AsyncSession, large_data_id: int, project_dir: str):
             with open(tarfile_name, "rb") as f:
                 while data := f.read(CHUNK_SIZE):
                     await lobject.write(data)
+
+
+async def import_project_file(db: AsyncSession, file, file_type, project_id):
+    directory, filename = file
+    full_path = os.path.join(directory, filename)
```

Comment:
The fact that you pass a tuple into a function to only join it indicates that tuple is not needed to be passed to a function at all.